stingray/crossspectrum.py

```diff
@@ -59,6 +61,13 @@ def __init__(self, lc1=None, lc2=None, norm='none'):
         norm: {'frac', 'abs', 'leahy', 'none'}, default 'none'
             The normalization of the (real part of the) cross spectrum.
 
+        Other Parameters
```

Comment:
Random question: why does `gti` get a separate category? Why is it not just with the other parameters?